camunda-transaction-boundaries-plugin/menu/menu.js

```diff
@@ -4,15 +4,7 @@ module.exports = function(electronApp, menuState) {
   return [{
     label: 'Toggle Transaction Boundaries',
     accelerator: 'CommandOrControl+t',
-    enabled: function() {
```

Comment:
Why this change? I liked the comments. I don't think we have to use arrow functions here

Reply:
I'm happy to revert, I just favor arrow functions for one liners.